src/odemis/model/test/remote_test.py

```diff
@@ -157,12 +158,17 @@ def test_mock(self):
 
 # @unittest.skip("simple")
 class ProxyOfProxyTest(unittest.TestCase):
-# Test sharing a shared component from the client
+    # Test sharing a shared component from the client
+
+    # @classmethod
+    # def setUpClass(cls) -> None:
+    #     testing.use_fake_backend_directory()
```

Comment:
why don't you add a test case that uses the fake backend directory or add a comment why it is unused here.

Reply:
Created test case for it... it turned out to be a really good idea, as there was a bug in the code!